### PR TITLE
HW 14 Done

### PR DESCRIPTION
**Pull request by OlesNikon**

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (landriiuk) [inline cypress.json:1]: U have empty basic configuration file, add here baseUrl as minimum

File: cypress/config_test_data/test_data.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import faker from 'faker';
+
+export const BASE_URL = "https://demoqa.com/text-box"
```
Comment (landriiuk) [inline cypress/config_test_data/test_data.js:3]: missed ";"

File: cypress/integration/task14.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    SUBMIT_BUTTON,
+    OUTPUT_USER_NAME,
+    OUTPUT_EMAIL,
+    OUTPUT_CURRENT_ADDRESS,
+    OUTPUT_PERNAMENT_ADDRESS
+} from "../config_test_data/test_data.js"
+
+describe("Populating and submitting the 'Text Box' form", () => {
+    before(() => {
+        cy.visit(BASE_URL)});
```
Comment (landriiuk) [inline cypress/integration/task14.js:20]: if u add basic config for the cypress.json u will write `cy.visit('/')`

File: package.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "function",
+  "version": "1.0.0",
+  "description": "aqa task",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```
Comment (landriiuk) [inline package.json:7]: U can delete this script

**Review by landriiuk — COMMENTED** [review]
Your code really good written, pls check the comments
Score 9,5/10

**Comment by landriiuk** [comment]
U can merge this branch into master

File: cypress/integration/task14.js (hunk 3 above)
Comment (OlesNikon) [reply cypress/integration/task14.js:20]: I fixed this in the next pull request)

**Review by landriiuk — APPROVED** [review]

**Pull request merged**